### DISCUSSION TITLE
Connection could not be established with host \"postal.qollega.id:25\": stream_socket_client(): Unable to connect to postal.qollega.id:25 (A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond)

### DISCUSSION THREAD

**Discussion by mrvisca** [postalserver/postal#2206] [Help with using Postal]
i have error when i tested send postal mail in local swagger, 
Is this normal or is there something wrong with my laravel smtp?

**Comment by willpower232** [comment]
It seems to be fine, you might need to watch the Postal SMTP logs to see what your code is doing.

Alternatively you might want to have a look at https://github.com/synergitech/laravel-postal if you're using laravel.
